<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clost-pair Caculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="3">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>388</width>
        <height>217</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>操作栏</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="3">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>53</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="pushButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>计算</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>生成数据</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>放大</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>缩小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>重置大小</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton</zorder>
      <zorder>horizontalSpacer_2</zorder>
      <zorder>horizontalSpacer</zorder>
      <zorder>horizontalSpacer_3</zorder>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>开启进度条</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>689</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>功能(&amp;D)</string>
    </property>
    <addaction name="actionGenerate"/>
    <addaction name="actionWork"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <widget class="QMenu" name="menu_Import">
     <property name="title">
      <string>导入(&amp;I)</string>
     </property>
     <addaction name="actionImportText"/>
     <addaction name="actionImportBinary"/>
    </widget>
    <widget class="QMenu" name="menu_Save">
     <property name="title">
      <string>保存(&amp;S)</string>
     </property>
     <addaction name="actionSaveText"/>
     <addaction name="actionSaveBinary"/>
    </widget>
    <addaction name="actionInput"/>
    <addaction name="menu_Import"/>
    <addaction name="menu_Save"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionInterface_Introduction"/>
    <addaction name="actionA"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置(&amp;W)</string>
    </property>
    <addaction name="actionNumber"/>
    <addaction name="actionRange"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSetting"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <action name="actionGenerate">
   <property name="text">
    <string>生成随机点(&amp;G)</string>
   </property>
  </action>
  <action name="actionWork">
   <property name="text">
    <string>计算最近点对(&amp;R)</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于本软件(&amp;A)</string>
   </property>
  </action>
  <action name="actionImportBinary">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>导入二进制数据(&amp;O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveBinary">
   <property name="text">
    <string>保存为二进制文件(&amp;B)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionNumber">
   <property name="text">
    <string>点的数量(&amp;P)</string>
   </property>
  </action>
  <action name="actionRange">
   <property name="text">
    <string>坐标范围(&amp;C)</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>关于QT(&amp;Q)</string>
   </property>
  </action>
  <action name="actionImportText">
   <property name="text">
    <string>导入文本数据(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSaveText">
   <property name="text">
    <string>保存为文本(&amp;T)</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>手动输入(&amp;U)</string>
   </property>
  </action>
  <action name="actionA">
   <property name="text">
    <string>功能介绍(&amp;E)</string>
   </property>
  </action>
  <action name="actionInterface_Introduction">
   <property name="text">
    <string>界面介绍(&amp;I)</string>
   </property>
  </action>
  <action name="actionPoint_tp">
   <property name="text">
    <string>绘制点设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
